src/SprykerSdk/Acp/Configuration/Reader/AppConfigurationReader.php

```diff
@@ -31,7 +33,7 @@ protected function readFile(string $filePath): string
         $content = file_get_contents($filePath);
 
         if ($content === false) {
-            return '';
+            throw new AppConfigurationException('Unable to read configuration file');
```

Comment:
Please add @throws annotation to the methods doc block.

Reply:
What is the reason to throw an exception here? How it worked before by passing an empty string. Is that a bug you found? Is the content really needed or is it maybe optional?

Reply:
That's a change @k4emic asked for here: https://github.com/spryker-sdk/acp/pull/43#discussion_r1004418022

here the content of that file is necessary to generate openapi schema for configuration

Reply:
Ok